manifests/init.pp

```diff
@@ -15,7 +15,6 @@
   $localhost6_aliases    = ['localhost6',
                             'localhost6.localdomain6'],
   $purge_hosts           = false,
-  $target                = '/etc/hosts',
```

Comment:
This would need to stay so that people may specify a target, though it should default to `undef` so that the provider can pick the correct one.